doozerlib/operator_metadata.py

```diff
@@ -52,7 +52,10 @@ def update_and_build(nvr, stream, runtime, merge_branch, force_build=False):
     :param string merge_branch: Which branch should be updated in the metadata repo
     :return bool True if operations succeeded, False if something went wrong
     """
-    op_md = OperatorMetadataBuilder(nvr, stream, runtime=runtime)
+    if runtime.group_config.vars['MAJOR'] == 4 and runtime.group_config.vars['MINOR'] == 2:
```

Comment:
really dislike anything version-specific in code. can we just extend the `operator_ref_arch` implementation from https://jira.coreos.com/browse/ART-839 and configure it with
```
operator_ref_arch: multichannel
```
(and take the enabled arch list from group.yml)